vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -116,6 +116,12 @@ export default {
           url: "/jobs",
           event: "tap_header_career",
         },
+        {
+          name: "About",
+          url: "/about",
+          className: Config.SHOW_ABOUT_US_PAGE ? "tw-visible" : "tw-hidden",
```

Comment:
On clicking it, underline is not visible like other menus
Also fix spacing in mobile devices